1-user.md

```diff
@@ -86,11 +86,13 @@ any yaml file at /etc/netplan: (change interface names, ip ranges accordingly)
        bridges:
          cloudbr0:
            addresses: [192.168.122.10/24]
-           gateway4: 192.168.122.1
+           routes:
+             - to: default
+               from: 192.168.122.1
            nameservers:
              addresses: [1.1.1.1,8.8.8.8]
            interfaces: [ens3]
-           dhcp4: false
+           dhcp4: true
```

Comment:
We don't need this if we want to setup static IP, `192.168.122.10/24` in the example.

Reply:
I am not able to access internet from inside the VM. Let me debug the exact issue and fix this.

Reply:
@vishesh92 it's possible you'll need to check/fix the static IP and gateway. Depending on whether this is a VM on KVM, VirtualBox, VMware workstation/fusion etc. some of those require the nic to be activated (which means sometimes restarting the VM or doing a ping to the gateway).